index.html

```diff
@@ -0,0 +1,146 @@
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale =1.0">
+        <title>Portfolio project</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/d901e0cb9a.js" crossorigin="anonymous"></script>
+    </head>
+    <body>
+      
+        <header>
+          <nav>
+            <ul>
+              <li><a class="logolink" href=""><b>My Logo</b></a></li>
```

Comment:
- [ ] You have created a good header but please move this logo out of the `nav` element. The logo should be inside the header but not inside the nav. The nav should then follow after the logo